components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/inbound/oauth2/OAuthConsumerAppToApiModel.java

```diff
@@ -112,6 +120,23 @@ private List<String> getAudiences(OAuthConsumerAppDTO oAuthConsumerAppDTO) {
         }
     }
 
+    private List<String> getIdTokenAudiences(OAuthConsumerAppDTO oAuthConsumerAppDTO) {
+
+        if (oAuthConsumerAppDTO.getIdTokenAudiences() == null) {
+            return Collections.emptyList();
+        } else {
+            return Arrays.asList(oAuthConsumerAppDTO.getIdTokenAudiences());
+        }
```

Comment:
can simplify this by using the ternary operator in java

Reply:
fixed with